src/components/facets/facet-check-list/check-list-item/facet-check-list-item.component.html

```diff
@@ -14,7 +14,7 @@
     [class.disabled]="facet?.disabled">
 
     <!-- Show check icon to indicate the state -->
-    <ux-checkbox [clickable]="false" [value]="selected" [simplified]="simplified" [tabindex]="-1" [disabled]="disabled">
+    <ux-checkbox [clickable]="false" [value]="selected" [simplified]="simplified" [tabindex]="-1" [disabled]="disabled" [id]="id">
```

Comment:
This will lead to duplicate ids because you've just given the same id to the `ux-facet-check-list-item`. I've asked you several times to check how the select component achieves this, but I guess you didn't so I'll just link it to you now:

https://github.com/UXAspects/UXAspects/blob/develop-1.8.18/src/components/select/select.component.html#L4
https://github.com/UXAspects/UXAspects/blob/develop-1.8.18/src/components/select/select.component.html#L25